Archiving.md

```diff
@@ -0,0 +1,63 @@
+# Archiving & Abandoning Repositories
+
+This is a checklist of tasks to complete for abandoning and archiving repositories.
+
+## Open a Pull Request with the following changes
+
+### Update the README
+
+- Remove any CI badges
+- Prepend a notice to the top of the README similar to the following passage:
+  ```markdown
+  > [!CAUTION]
+  > This package is **abandoned** and will receive no further development.
+  >
+  > See the Technical Steering Committee [meeting minutes](url to the minutes).
+  >
+  > Optionally a sentence about an actively maintained alternative library.
+  ```
+- Check for and remove any content from the README that is now irrelevant.
+
+### Update `composer.json`
+
+Add the `"abandoned":true` to `composer.json` at the top-level of the file and run `composer update nothing` to refresh the lockfile.
+
+### Remove `renovate.json`
+
+Remove the renovate configuration file - this is possibly unnecessary, as once archived, renovate is unlikely to open new PRs.
+
+### Update `mkdocs.yml`
+
+In `mkdocs` configuration, alter the `site_name` variable, for example:
+
+```diff
+- site_name: Some Name
++ site_name: Some Name (Abandoned)
+```
+
+## Close all existing PRs and Issues
+
+Once the pull request has been merged, close all open issues and PRs, and ideally, leave a comment with a reason for the closure such as: "This library/project has been abandoned".
```

Comment:
Why is this necessary? Archiving the repo already prohibits new replies :thinking:

Reply:
My opinion is that it's decent to let whomever opened the PR/Issue know that it's not getting fixed/merged. I don't think it's _necessary_ per se